qml.qrc

```diff
@@ -213,6 +213,16 @@
         <file>images/restore-wallet-from-hardware.png</file>
         <file>images/open-wallet-from-file@2x.png</file>
         <file>images/open-wallet-from-file.png</file>
```

Comment:
```suggestion
```

Reply:
This icon is still being used in "Open wallet from file" main menu item.

Reply:
sorry

<img width="571" alt="Screenshot 2021-11-29 at 20 09 44" src="https://user-images.githubusercontent.com/7697454/143927998-33319def-b2e3-44e1-83c2-cc75578a8291.png">

looked like you deleted the file :) ignore my comment